Chirp/Model/User.swift

```diff
@@ -6,7 +6,10 @@
 //
 
 import Foundation
+import UIKit
 
 struct User {
     var name: String?
+    var phoneNumber: String?
+    var profilePicture: UIImage?
```

Comment:
Can we make this a String instead? Because it will be a URL that will come from the backend